#goal definition
trajectory_msgs/JointTrajectory JointTargets
---
#result definition
bool Success
---
#feedback
# Command State Definitions
uint8 JTA_READY=0
uint8 JTA_TRANS_EULER=1
uint8 JTA_TRANS_QUAT=2
uint8 JTA_TRAJ=3

# Grasping State Definitions
uint8 JTA_GRASP_LIMP=0
uint8 JTA_GRASP_STATIC=1
uint8 JTA_GRASP_NOW=2
uint8 JTA_GRASP_AT_END=3
uint8 JTA_GRASP_RELEASE_NOW=4
uint8 JTA_GRASP_RELEASE_AT_END=5

# Error State Definitions
uint8 JTA_NO_ERROR=0
uint8 JTA_INVALID_POSE=1
uint8 JTA_INVALID_TRANSITION=2
uint8 JTA_BROKEN_CHAIN=3

uint8 CommandState
uint8 GraspState
uint8 ErrorState
